monitoring/uss_qualifier/suites/faa/uft/design/README.md

```diff
@@ -0,0 +1,87 @@
+# UFT Message Signing tests for SCD using Uss_Qualifier
+
+UTM Field Test (UFT) has a requirement for USSes to sign their message requests
+and responses in the SCD flow using IATF certificates provided by FAA.
+
+This test suite helps to test the message signing by extending [suites.astm.utm.f3548_21](../../../astm/utm/f3548_21.yaml)
+with scenarios that trigger reporting of message signing in the interactions with mock_uss.
+
+## Test Setup
+
+The test setup includes running the following components on your local machine, as also shown in the [diagram](./InterUss_Test_Harness_With_Message_Signing.png)
+
+1. Uss Qualifier - This is the test driver that injects the operations for the SCD flow tests in USSes.
+2. Auth - This auth server provides access tokens and the public key for token validation for USS-to-USS and USS-to-DSS communication. In a local deployment of the test infrastructure, this can be supplied by an instance of dummy auth running as a dockerized container exposing port 8085, as per build/dev/run_locally.sh.
+3. DSS - This is a local DSS running in dockerized container, and is available at port 8082.
```

Comment:
Along the same lines of [the comment for auth](https://github.com/interuss/dss/pull/899#discussion_r1023160627), could this be generalized as well?

```suggestion
3. DSS - This is a DSS instance supporting SCD in the UTM environment defined by the auth server.  In a local deployment of the test infrastructure, this can be supplied by the local DSS instance running as a dockerized container exposing port 8082, as per build/dev/run_locally.sh.
```